### DISCUSSION TITLE
ID resolution process

### DISCUSSION THREAD

**Discussion by syncronox** [merchantprotocol/theadslab#37] [Open Discussion Forum]
![Pixel Fire](https://user-images.githubusercontent.com/103632406/163495277-3e559cb8-c679-4836-bc85-dfc5b09d5cf6.png)
Is this the general process? 

Pixel fires-hashed devices-consumer emails-then it matches to their profiles

**Comment by jonathonbyrdziak** [comment]
You didn't allow your image to finish uploading before posting your comment. please try again

**Reply by syncronox** [reply]
Ok just did
